src/Sylius/Bundle/AdminBundle/templates/taxon/sections/tree/actions/dropdown/delete.html.twig

```diff
@@ -0,0 +1,37 @@
+{# Rendered with \Sylius\Bundle\AdminBundle\Twig\Component\Taxon\DeleteComponent #}
+
+{% import '@SyliusAdmin/shared/helper/button.html.twig' as button %}
+{% set id = taxon_id|default('__TAXON_ID__') %}
```

Comment:
Is this default value necessary?

Reply:
Yes, because it is rendered in `infiniteTree`, and replaced by the ID of taxon. When we want to delete a taxon, the `deleteComponent` has the information about the taxon ID.